### DISCUSSION TITLE
Specifying Path to point to /src

### DISCUSSION THREAD

**Discussion by mpaustin1993** [ElMassimo/vite-plugin-full-reload#4] [Q&A]
Hello! Great plugin - thank so much for implementing this.

I'm currently running a Vue3 app & I'm trying to have the app fully reload upon any edits to all Vue components in my views folder:

Is this the proper configuration?

`FullReload(['views/**/*.*', 'src/'])`

**Comment by ElMassimo** [comment]
Hi Michael! @mpaustin1993

Are you using a backend integration or just plain Vite?

Typically, if you are only using Vue 3, the `@vitejs/plugin-vue` plugin would perform HMR when you change any Vue file, so you would not need to reload the entire page, it would re-render or re-mount that component only (great DX).

Can you share the tree structure of your project? Is the `views` dir outside `src`?

**Reply by mpaustin1993** [reply]
@ElMassimo Maximo!

Thanks for getting back to me so quickly - didn't even know the Discussions tab existed so I'll use that in the future.

I'm using plain Vite w/ Vue3 - HMR works well for most functionality, but not everything (child components) so I've been looking for a full-reload functionality similar to what `vue-cli-service` offers (not even sure if that fully re-loaded).

Tree structure: `src/views/<page folder>/<component>.vue`

**Reply by ElMassimo** [reply]
Gotcha, then you should adjust the config:

```ts
FullReload(['src/views/**/*'])
```

All watched paths will be [resolved using the project root](https://github.com/ElMassimo/vite-plugin-full-reload/blob/main/src/index.ts#L49).

---

As an aside, it's odd that HMR would not work for certain components, you might want to report that to vuejs/core.

**Reply by mpaustin1993** [reply]
I tried that path in the config path before, but I'll give it another shot.

I'm using Element-Plus for most of my components' elements, which hasn't always been the most responsive - if the issue comes up again, I'll report it vuejs/core.

Thanks for all of your help Maximo.
